test/e2e/utils_test.go

```diff
@@ -118,38 +119,41 @@ func getTestManifest(t *testing.T, baseManifestPath string) io.Reader {
 	manifestsReader, err = os.Open(baseManifestPath)
 	require.NoError(t, err)
 
-	manifestsReader, err = patchControllerImageFromEnv(t, manifestsReader)
-	if err != nil {
-		t.Logf("failed patching controller image (%v), using default manifest %v", err, baseManifestPath)
-		return manifestsReader
-	}
+	if !skipTestPatches {
+		t.Logf("applying test patches to manifest %v", baseManifestPath)
 
-	manifestsReader, err = patchGatewayImageFromEnv(t, manifestsReader)
-	if err != nil {
-		t.Logf("failed patching gateway image (%v), using default manifest %v", err, baseManifestPath)
-		return manifestsReader
-	}
+		manifestsReader, err = patchControllerImageFromEnv(t, manifestsReader)
```

Comment:
Are the spaces redundant?

Reply:
I don't think so, it's an indentation that was introduced for `if !skipTestPatches {}` block.